pkg/cmd/step/step_changelog.go

```diff
@@ -205,28 +205,8 @@ func (o *StepChangelogOptions) Run() error {
 		o.BatchMode = true
 	}
 
-	apisClient, err := o.ApiExtensionsClient()
```

Comment:
how odd. Why was this ever here?

Reply:
It's not the first place where I've seen it. It's legacy code added before we started registering CRDs on install / boot.